File: paquete_python/paquete_python/parametros.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node

class Clase_Nodo(Node):

    def __init__(self):
        super().__init__("nombre_parametros_python")    
        
        # Declarar parámetro (nombre, valor por defecto)
        self.declare_parameter("nombre_parametro", "valor_por_defecto_py")

        # Obtener parámetro
        valor = self.get_parameter("nombre_parametro").value

        # Mostrarlo
        self.get_logger().info(f"nombre_parametro = '{valor}'")
        
def main(args=None):
    rclpy.init(args=args)
    
    objeto_nodo = Clase_Nodo()
    rclpy.spin(objeto_nodo)
    
    rclpy.shutdown()

if __name__ == '__main__':
    main()

File: legacy/paquete_cpp/src/parametros.cpp
#include "rclcpp/rclcpp.hpp"

class Clase_Nodo : public rclcpp::Node
{
    public:
        Clase_Nodo() : rclcpp::Node("nombre_parametros_cpp")
        {
            // Declarar parámetro (tipo, nombre, valor por defecto)
            this->declare_parameter<std::string>("nombre_parametro", "valor_por_defecto_cpp");

            // Obtener parámetro (decir tipo)
            std::string valor = this->get_parameter("nombre_parametro").as_string();

            // Mostrarlo
            RCLCPP_INFO(this->get_logger(), "nombre_parametro = '%s'", valor.c_str());
        }
    private:
};

int main(int argc, char * argv[])
{
    rclcpp::init(argc, argv); 

    auto objeto_nodo = std::make_shared<Clase_Nodo>();
    rclcpp::spin(objeto_nodo);

    rclcpp::shutdown();
    return 0;
}